Exiled.API/Features/Map.cs

```diff
@@ -31,6 +31,18 @@ public static class Map
         private static readonly List<Camera079> CamerasValue = new List<Camera079>(250);
         private static readonly List<Lift> LiftsValue = new List<Lift>(10);
         private static readonly List<TeslaGate> TeslasValue = new List<TeslaGate>(10);
+        private static readonly Lazy<Ragdoll.Info> LazyDefaultRagdollOwner = new Lazy<Ragdoll.Info>(() =>
+        {
+            return new Ragdoll.Info()
+            {
+                ownerHLAPI_id = PlayerManager.localPlayer.GetComponent<Dissonance.Integrations.MirrorIgnorance.MirrorIgnorancePlayer>().PlayerId,
```

Comment:
What will cause null/empty string as a value?

Reply:
I couldn't test it, and I'm also wondering what will happen. Either reject it because it requires more testing or just say this is almost absolutely untested and might break between rounds; either way the overload Map::SpawnRagdoll that uses the ev.Player should work flawlessly.

Reply:
Will test it for you.

Reply:
Ok, the test was successful. Nothing happens if I use null/invalid id.

Reply:
Marking it as resolved, many thanks!